lib/generamba/cli/gen_command.rb

```diff
@@ -58,5 +61,24 @@ def gen(module_name, template_name)
       generator.generate_module(module_name, code_module, template)
     end
 
+    no_commands {
+      def print_parameters_in_table(parameters, module_name)
+        params = parameters.clone
+
+        templates = []
+
+        params["templates"].each do |param|
```

Comment:
OK, I see. Another possible way is to introduce a new class, TableParametersFormatter, which will be responsible for preprocessing data before printing.